src/components/app/App.js

```diff
@@ -1,22 +1,12 @@
 import React from 'react';
-import './App.css';
-import Board from '../board/board'
-import { connect } from 'react-redux';
-
-const  App = () => {
-
-  return (
-    <div>
-      <Board /> 
-    </div>
-  );
+import Board from '../board/board';
```

Comment:
Nice! I think this will help as the app is added onto, such as bringing in players, etc.